static/src/App.vue

```diff
@@ -28,13 +28,13 @@ export default defineComponent({
       <h1 class="_text-align:center">Annoto</h1>
     </i-layout-header>
     <i-layout-content>
-      <ImageDisplay :src="apiUrl + imageUrl" />
+      <ImageDisplay :src="apiUrl + 'images/' + imageSrc + '?'" />
```

Comment:
Maybe it would make sense to use the generated `getUrl` function here, if that is possible? Im not sure if the parameters are available

Reply:
Better now?